internal/statemachine/classic_states.go

```diff
@@ -581,10 +581,48 @@ func (stateMachine *StateMachine) installPackages() error {
 	var installPackagesCmds []*exec.Cmd
 
 	// mount some necessary partitions from the host in the chroot
-	mounts := []string{"/dev", "/proc", "/sys"}
+	type mountPoint struct {
+		dest     string
+		fromHost bool
+	}
+	mountPoints := []mountPoint{
+		{
+			dest:     "/dev",
+			fromHost: true,
+		},
+		{
+			dest:     "/proc",
+			fromHost: true,
+		},
+		{
+			dest:     "/sys",
+			fromHost: true,
+		},
+		{
+			dest:     "/run",
+			fromHost: false,
+		},
+	}
+
 	var umounts []*exec.Cmd
-	for _, mount := range mounts {
-		mountCmd, umountCmd := mountFromHost(stateMachine.tempDirs.chroot, mount)
+	for _, mount := range mountPoints {
+		var mountCmd, umountCmd *exec.Cmd
+		var err error
```

Comment:
nitpick: the `error` variable is only used inside the `else` branch, so we could move its declaration down there to keep it close to its usage.